editoast/Dockerfile

```diff
@@ -28,6 +29,8 @@ RUN rustup component add llvm-tools && \
     rustup component add rustfmt && \
     rustup component add clippy && \
     cargo install grcov
+RUN curl --location --output /tmp/cargo-tomlfmt.tar.gz https://github.com/tbrand/cargo-tomlfmt/releases/download/v0.1.4/cargo-tomlfmt-v0.1.4-x86_64-unknown-linux-musl.tar.gz
+RUN tar --extract --verbose --gzip --file /tmp/cargo-tomlfmt.tar.gz --directory /usr/local/bin/ cargo-tomlfmt
```

Comment:
Can't we use `cargo install cargo-tomlfmt`?

Reply:
We can, it's just longer to install, and I don't see the value if there is a released artifact (readability maybe 🤔). Do you think it doesn't matter ? (maybe the Docker layering will make it run only once, and therefore, "longer to install" is not really a problem ?).

For readability, we could use [`cargo binstall`](https://github.com/cargo-bins/cargo-binstall) which would do the same thing (check if there is released on Github, and install from it if it exists), but then, we have to install `cargo binstall` first 😄

Reply:
It was for readability and consistency with `diesel-cli` but I'm okay keeping it this way.

Reply:
Ok, finally, we're going to use a non-released version as discussed, so switching back to `cargo install`.